_ap/17/index.adoc

```diff
@@ -0,0 +1,62 @@
+---
+num: 17
+tags:
+  - "tls"
+  - "certificates"
+  - "dns"
+  - "Letsencrypt"
+title: "Automatic management of certificates per data plane cluster using Letsencrypt"
+status: "Accepted"
+authors:
+  - "Manyanda Chitimbo"
+---
+
+## Intent
+
+Define a pattern for the automatic management of certificates per data plane cluster using Letsencrypt
+
+## Motivation
+
+When a managed service is deployed on the data plane cluster its routes have be reachable via TLS.
+
+It means that a managed service a way to the clients to verify and validate the TLS connection with the related handshake with the service exposed outside of the data plane cluster.
```

Comment:
I did not understand this sentence, the beginning.

Reply:
Ah reading this again and I feel like something might have slipped out before I pushed the changes. I'll revisit this.
Thanks for the review.